### PR TITLE
Stable/newton

### PR DESCRIPTION
**Pull request by zancas**
@pjbreaux this is a merge forward from mitaka

### CODE REVIEW

**Comment by zancas** [comment]
Here's a link to the tests demonstrating success:

http://jenkins.pdbld.f5net.com/job/openstack/job/driver/job/newton/job/za_test_11.6.1_undercloud_vxlan/7/console

**Comment by pjbreaux** [comment]
Can one of the admins verify this patch?

**Comment by pjbreaux** [comment]
Can one of the admins verify this patch?

File: f5lbaasdriver/test/tempest/Docker/tempest_tests [hunk 1]
```diff
@@ -12,8 +12,8 @@ RUN apt-get -y update \
 RUN pip install --upgrade pip
 RUN pip install tempest
 RUN pip install crudini
-RUN pip install git+https://github.com/openstack/neutron.git@stable/mitaka
-RUN pip install git+https://github.com/openstack/neutron-lbaas.git@stable/mitaka
+RUN pip install git+https://github.com/openstack/neutron.git@mitaka-eol
```
Comment (szakeri) [inline f5lbaasdriver/test/tempest/Docker/tempest_tests]: I am not sure where is docker file is used, if it is used in the driver, then this needs to change to netwon branch.

File: systest/Makefile [hunk 2]
```diff
@@ -183,6 +185,7 @@ tempest_12.1.1_overcloud_vxlan:
 	export GUMBALLS_PROJECT=$(CHANGESOURCE)_$(SANITIZED_BRANCH)-$@ ;\
 	export RESULTS_DIR=$(MAKEFILE_DIR)/test_results/$${GUMBALLS_PROJECT} ;\
 	export TEST_SESSION=$@_$(SANITIZED_BRANCH)_$(SUBJECTCODE_ID)_$(TIMESTAMP) ;\
+	export TEST_TENANT_NETWORK_TYPE=vxlan ;\
```
Comment (szakeri) [inline systest/Makefile]: "TEST_TENANT_NETWORK_TYPE" needs to be set for all of the build targets, it is not used in the overcloud tlc file, however it is used in the systest/scripts/ansible_install_lbaasv2.sh

**Comment by zancas** [comment]
@pjbreaux your button await-eth in its most green state.

**Comment by pjbreaux** [comment]
@zancas: Please provide a full commit message.

**Comment by zancas** [comment]
@pjbreaux look!

**Pull request merged**